### PR TITLE
Specific vertion of PS where it is available

### PR DESCRIPTION
**Pull request by msaustral**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.x / 8.x
| Description?  | Please be specific when describing the PR. What did you add, why did you submit it?
| Fixed ticket? | Fixes #{issue number here} if there is a related issue

### CODE REVIEW

File: development/components/form/types-reference/number-min-max-filter.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+ONLY AVAILABLE FROM PS 1.7.7.X ->
```
Comment (thomasnares) [inline development/components/form/types-reference/number-min-max-filter.md:1]: We should not indicate this information here, but in the first heading of the page, with the specific markdown

```
{{< minver v="1.7.7" >}}
```

**Review by thomasnares — COMMENTED** [review]
Hi @msaustral, thanks for your contribution, i left you a comment 👍 

Can you put it on the H1 line of the page (line 7)

**Review by kpodemski — CHANGES_REQUESTED** [review]
Hello @msaustral 

Please follow @thomasnares suggestions.

**Comment by kpodemski** [comment]
We have this information in ps8 docs, I think we can close this one.

**Pull request closed without merging**